docs/scripting-doc/index.d.ts

```diff
@@ -4328,4 +4333,9 @@ declare class Dialog {
    * the provided width, it will scale itself up automatically.
    */
   minimumHeight: number;
+  /**
+   * Set the style sheet for the entire dialog.
+   * @param styleSheet The text of your style sheet. See https://doc.qt.io/qt-6/stylesheet.html and https://doc.qt.io/qt-6/stylesheet-examples.html for more information
+   */
+  setStyleSheet(styleSheet: string) : void;
```

Comment:
Rather than documenting `setStyleSheet` here, I think we could derive `Dialog` from `Qt.QWidget`. The `minimumWidth` and `minimumHeight` properties could be moved there as well, since they're `QWidget` properties after all and should work on all widgets you can add to the dialog as well.

Reply:
Oh, btw, I think this should just be documented as a `styleSheet : string` property.

Reply:
Done thanks